airflow/www/templates/airflow/graph.html

```diff
@@ -128,6 +128,7 @@
     const edges = {{ edges|tojson }};
     const tasks = {{ tasks|tojson }};
     let taskInstances = {{ task_instances|tojson }};
+    const autoRefreshInterval = {{ auto_refresh_interval }};
```

Comment:
Dynamically generated javascript is risky. Can you use `getMetaValue` method instead?

Reply:
It's not risky in this case as the value comes from a trusted source, not user input.